bundles/org.openhab.binding.telegram/README.md

```diff
@@ -196,27 +196,27 @@ These actions will send a message to all chat ids configured for this bot.
 | Action                     | Description  |
 |----------------------------|--------------|
 | sendTelegram(String message) | Sends a message. |
-| sendTelegram(String format, Object... args)          | Sends a formatted message (See https://docs.oracle.com/javase/8/docs/api/java/util/Formatter.html for more information).
+| sendTelegram(String format, Object... args)          | Sends a formatted message (See <https://docs.oracle.com/javase/8/docs/api/java/util/Formatter.html>
```

Comment:
End parenthesis missing. "for more information" deleted on purpose?

Reply:
Nope. Not on Purpose.